portal-api/api-tests/v1/bank-holidays/bank-holidays.api-test.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @jest-environment node
+ */
+
+const app = require('../../../src/createApp');
+const testUserCache = require('../../api-test-users');
+const { withClientAuthenticationTests } = require('../../common-tests/client-authentication-tests');
+const { withNoRoleAuthorisationTests } = require('../../common-tests/role-authorisation-tests');
+
+const { as, get } = require('../../api')(app);
+
+jest.unmock('../../../src/external-api/api');
```

Comment:
Is this here explicitly because other tests are mocking external-api and you want to ensure it's not mocked for this test? I'm not too familiar with `jest.unmock` but a quick read seems to suggest that the use case is for when you want to mock certain parts of an API but not others and so I don't know if it's needed here

Reply:
I hadn't thought too much about this and was just unmocking the API as that's what they do in another API test, after a bit more consideration I agree that it's not suitable to unmock the external-api here so I'll update the tests